script/hassfest/translations.py

```diff
@@ -532,6 +532,11 @@ def validate_translation_file(  # noqa: C901
                 "translations",
                 f"{reference['source']} contains invalid reference {reference['ref']}: Could not find {key}",
             )
+        elif match := re.match(RE_REFERENCE, search[key]):
+            integration.add_error(
+                "translations",
+                f"Lokalise supports only one level of references: \"{reference['source']}\" should point to directly to \"{match.groups()[0]}\"",
```

Comment:
I think this line is too long

Reply:
This file contains even longer lines (https://github.com/home-assistant/core/blob/73a9836c91cc6174d95a3dbe7fe65c4681eee60f/script/hassfest/translations.py#L401)
and also, Line 533 is longer than 88 characters.

For consistency, I would leave it as it is.

If we enforce character limits in this file, we should refactor the whole file in another PR.